spec/player_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../lib/board'
+require_relative '../lib/player'
+
+RSpec.describe Player do
+  let(:board) { Board.new(3) }
+  let(:player) { Player.new('Bob', 'X') }
+  it 'initializes a player with name and symbol and gives them a unique id' do
```

Comment:
I think the unique id part can be removed from the test description now?

Reply:
Resolved at ff95fbb6e155906d4c2cd1434215bdaa182646ac

Reply:
Resolved at ddb7c97cdf62bf59a2f5f0938465b319f30a75bc